network-store-integration-test/src/test/java/com/powsybl/network/store/integration/NetworkStoreIT.java

```diff
@@ -585,6 +589,53 @@ public void testGeneratorCurveReactiveLimits() {
         }
     }
 
+    @Test
+    public void testBusBreakerNetwork() {
+        try (NetworkStoreService service = new NetworkStoreService(getBaseUrl())) {
+            service.flush(BusBreakerNetworkFactory.create(service.getNetworkFactory()));
+        }
+
+        try (NetworkStoreService service = new NetworkStoreService(getBaseUrl())) {
+
+            Map<UUID, String> networkIds = service.getNetworkIds();
+            assertEquals(1, networkIds.size());
+
+            Network readNetwork = service.getNetwork(networkIds.keySet().stream().findFirst().get());
+            ArrayList<Bus> buses = new ArrayList<>();
+
+            readNetwork.getBusBreakerView().getBuses().forEach(buses::add);
```

Comment:
You can do that using getBusStream in a cleaner and concise way.

Reply:
The whole point is to test getBuses there